modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -56,7 +57,8 @@ public static long addListTypeDefinition(
 		if (Validator.isNull(
 				objectField.getListTypeDefinitionExternalReferenceCode())) {
 
-			return 0;
+			objectField.setListTypeDefinitionExternalReferenceCode(
+				String.valueOf(UUID.randomUUID()));
 		}
```

Comment:
@mateussgn what happens if the `objectField` is not a picklist? We are not checking the business type [here](https://github.com/liferay/liferay-portal/blob/ac9e0db2bda89b01a466eb79af1a21d2810bba60/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L180-L184) or [here](https://github.com/liferay/liferay-portal/blob/ac9e0db2bda89b01a466eb79af1a21d2810bba60/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L211-L215).